src/index.js

```diff
@@ -1,8 +1,27 @@
 import _ from 'lodash';
 import './style.css';
 import createTask from './modules/createTask.js';
-import { mainTitle } from './modules/elements.js';
+import addTask from './modules/addTask.js';
+import { mainTitle, addTaskInput, btnEnter } from './modules/elements.js';
+
+let dataTask = [];
 
 mainTitle.innerHTML = _.join(["Today's", 'to do'], ' ');
-// crea tasks list//
-createTask();
+
+// retrieve tasks data and display the task list
+window.onload = () => {
+  if (localStorage.getItem('dataTasks') !== 'undefined' || localStorage.getItem('dataTasks') !== 'null') {
+    dataTask = JSON.parse(localStorage.getItem('dataTasks'));
+    createTask(0, dataTask.length, dataTask);
+  }
+};
```

Comment:
- [ ] Great job so far @ivangonzalez224 there are just a few changes I will suggest you make and you'll be on your way to the next milestone :wink: 
- [ ] When I run your project, there was an error that was preventing any of the functionalities you were to implement in this milestone from working at all. I later found out that the initial error was coming from this part of your code so kindly fix this so your app can run seamlessly and work as expected without any errors :+1: 
- [ ] I tried to fix the error so I could test if your functionalities were working so I edited this part of your code to look something like this after which the error disappeared :
```javascript
window.onload = () => {
  const storedData = localStorage.getItem('dataTasks');
  if (storedData !== null) {
    dataTask = JSON.parse(storedData);
    createTask(0, dataTask.length, dataTask);
  }
};
```
- Error Message
![image](https://github.com/ivangonzalez224/To-do-list/assets/102699958/9958f16d-d1e2-44e5-8f15-436b8d41632e)